src/main/resources/init_db.sql

```diff
@@ -0,0 +1 @@
+USE init_db;
```

Comment:
It's not enough , but mentors will tell you exactly, cz maybe it's also a valid way, but I don't think so

Reply:
Absolutely agree! Let’s create all scripts ;)